style.css

```diff
@@ -507,3 +518,157 @@ section {
   background-color: #fff;
   border-radius: 8px;
 }
+
+@media only screen and (min-width: 768px) {
+  body {
+    overflow-x: hidden;
+  }
+
+  #menu {
+    display: none;
+  }
+
+  .nav-links {
+    display: flex;
+    visibility: visible;
+    gap: 32px;
+    position: relative;
+    bottom: 15px;
+    right: 10.76%;
+  }
+
+  .nav-links a {
+    width: 46px;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .navigation,
+  .bio-section,
+  .about-section,
+  .contact-me {
+    width: 100vw;
+    overflow-x: hidden;
+  }
+
+  .bio-section {
+    height: 100vh;
+    position: relative;
+    top: -80px;
+    background-image: url(img/desk-bg.png);
+    background-size: 100% auto;
+  }
+
+  .bio-section .content {
+    padding: 10%;
+    width: 80vw;
+    position: relative;
+    top: 4%;
+  }
+
+  .bio-header {
+    width: 44vw;
+  }
+
+  .bio-text {
+    font-size: 1rem;
+  }
+
+  .cards {
+    display: flex;
+    flex-direction: column;
+    gap: 88px;
+  }
+
+  .card {
+    width: 80.2vw;
+    height: 70%;
+    position: relative;
+    left: 9.86%;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .card-img img {
+    height: 172%;
+    width: 102%;
+    position: relative;
+    top: -39%;
+    left: -5%;
+  }
+
+  .card-items {
+    width: 99%;
+    height: 81%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 12%;
+    position: relative;
+    left: 4%;
+    top: -16%;
+  }
+
+  .card-props {
+    margin-right: 7%;
+  }
+
```

Comment:
kindly fix the following parts I highlighted with a red marker as it looks far from the design given 😄 

Figma design :
![image](https://github.com/josman227/Mobile-Portfolio/assets/20537712/f0c1e35e-95c6-4346-afd5-1ab5574edca3)

Your design:
![image](https://github.com/josman227/Mobile-Portfolio/assets/20537712/f8b3cc67-9fe9-48a2-bd27-7e742d18d968)

Also, kindly use the right images ,text and alighnment of the cards for each card in your works section as shown in the figma template instead of using the same images and texts throughout

Your design :
![image](https://github.com/josman227/Mobile-Portfolio/assets/20537712/8546f683-1a40-42e5-a20f-cb4e47a3e7b5)

Figma design :
![image](https://github.com/josman227/Mobile-Portfolio/assets/20537712/fb76be9b-28b2-439c-80ae-52b0522a0870)